bundles/org.openhab.core/src/main/java/org/openhab/core/common/registry/AbstractRegistry.java

```diff
@@ -249,7 +249,7 @@ public void removed(Provider<E> provider, E element) {
             }
             Provider<E> elementProvider = elementToProvider.get(existingElement);
             if (elementProvider != null && !elementProvider.equals(provider)) {
-                logger.error(
+                logger.debug(
```

Comment:
Why is a failed removal no longer an error?

Reply:
Done